src/pages/class/[cid]/index.tsx

```diff
@@ -180,6 +180,12 @@ const InformationContainer = styled.div`
   grid-template-columns: auto 1fr auto;
   gap: 40px;
   box-shadow: 0px 0px 16px rgba(40, 40, 40, 0.12);
+  @media (max-width: 570px) {
```

Comment:
change this too.

Reply:
could you add a screenshot of how this changes the view?

Reply:
> change this too.

Updated here https://github.com/kixlab/kuiz/pull/82/commits/a57bc9ea68f6c46b4a6fd4f4d306b6a7ef5b8941 :)

> could you add a screenshot of how this changes the view?

Here are the before and after screenshots :)

![before](https://user-images.githubusercontent.com/112761202/236713171-80a315cc-1ad2-4696-8ecf-51b3a185d660.PNG)


![after](https://user-images.githubusercontent.com/112761202/236713205-14323ce7-82f2-4a2b-b312-b0d2734968ef.PNG)